openapi/index.yaml

```diff
@@ -1405,3 +1441,30 @@ definitions:
         $ref: '#/definitions/OrganizationFiscalCode'
     required:
       - fiscal_code
+  MessageCategory:
+    x-one-of: true
+    allOf:
+      - $ref: '#/definitions/MessageCategoryPayment'
+      - $ref: '#/definitions/MessageCategoryBase'
+  MessageCategoryBase:
+    type: object
+    properties:
+      tag:
+        type: string
+        enum:
+          - GREEN_PASS
```

Comment:
In a separate PR @fabriziofff pointed out that `GREEN_PASS` is not what we use normally: `EU_COVID_CERT` is used through the ecosystem instead.

Reply:
Changed the enum value in commons. I'll release it and port it here.